linera-core/src/unit_tests/test_helpers.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
```

Comment:
These `#[cfg(test)]` attributes are redundant because the grand-parent module already only includes the unit tests module only for testing:

https://github.com/SantiSjp/linera-protocol/blob/396f7e6949cfab8269af12ff3c98f5f459c7d160/linera-core/src/client/mod.rs#L89

Therefore, the attributes in this file should be removed.